internal/infrastructure/storage/db/postgres/sqlc/query.sql

```diff
@@ -4,36 +4,39 @@ INSERT INTO wallet(id, encrypted_mnemonic,password_hash,birthday_block_height,ro
 VALUES($1,$2,$3,$4,$5,$6,$7) RETURNING *;
 
 -- name: GetWalletAccountsAndScripts :many
-SELECT w.id as walletId,w.encrypted_mnemonic,w.password_hash,w.birthday_block_height,w.root_path,w.network_name,w.next_account_index, a.name,a.index,a.xpub,a.derivation_path as account_derivation_path,a.next_external_index,a.next_internal_index,a.fk_wallet_id,asi.script,asi.derivation_path as script_derivation_path,asi.fk_account_name FROM
+SELECT w.id as walletId,w.encrypted_mnemonic,w.password_hash,w.birthday_block_height,
+w.root_path,w.network_name,w.next_account_index, a.namespace,a.index,a.xpub,a.derivation_path as account_derivation_path,
+a.next_external_index,a.next_internal_index,a.fk_wallet_id, a.label, asi.script,asi.derivation_path as script_derivation_path,
+asi.fk_account_namespace FROM
 wallet w LEFT JOIN account a ON w.id = a.fk_wallet_id
-LEFT JOIN account_script_info asi on a.name = asi.fk_account_name
+LEFT JOIN account_script_info asi on a.namespace = asi.fk_account_namespace
 WHERE w.id = $1;
 
 -- name: UpdateWallet :one
 UPDATE wallet SET encrypted_mnemonic = $2, password_hash = $3, birthday_block_height = $4, root_path = $5, network_name = $6, next_account_index = $7 WHERE id = $1 RETURNING *;
 
 -- name: GetAccount :one
-SELECT * FROM account WHERE name = $1;
+SELECT * FROM account WHERE namespace = $1;
```

Comment:
Should be like this to support namespace/label, shouldn't it?
```suggestion
SELECT * FROM account WHERE namespace = $1 OR label = $1;
```

Reply:
This change should be made to all queries referring account only via namespace so far

Reply:
This is not possible without bigger change(using joins), I only updated GetAccount and used it to fetch account and use its namespace to get data from other tables.

Done